cmd/access/node_builder/staked_access_node_builder.go

```diff
@@ -67,7 +67,7 @@ func (fnb *StakedAccessNodeBuilder) InitIDProviders() {
 	})
 }
 
-func (builder *StakedAccessNodeBuilder) Initialize() cmd.NodeBuilder {
+func (builder *StakedAccessNodeBuilder) Initialize() error {
```

Comment:
Yup, the added bonus being that this makes the side-effecting behavior of the initialization explicit.